src/t3311/CMakeLists.txt

```diff
@@ -17,5 +17,8 @@ if (MODBUS_FOUND)
     if (BUILDSWIGPYTHON)
       swig_link_libraries (pyupm_${libname} ${MODBUS_LIBRARIES} ${PYTHON_LIBRARIES} ${MRAA_LIBRARIES})
     endif()
+    if (BUILDSWIGJAVA)
+      swig_link_libraries (javaupm_${libname} ${MODBUS_LIBRARIES} ${MRAA_LIBRARIES} ${JAVA_LIBRARIES})
```

Comment:
@petreeftime updated the linker flags in #328, shouldn't they be `${MRAAJAVA_LDFLAGS} ${JAVA_LDFLAGS}` now?